compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -468,10 +468,12 @@
 		fmins,                                                  	/* 0x1E205800	FMIN      	 */
 		fmind,                                                  	/* 0x1E605800	FMIN      	 */
 /* Internal OpCodes */
+		nop,   // nop                                           	/* 0xD503201F   NOP          */
```

Comment:
Is `nop` an internal opcode?

Reply:
I added `Hint instructions` section and moved `nop` to that section.